src/blr/cli/clusterrmdup.py

```diff
@@ -3,594 +3,303 @@
 sequences for reads sharing the duplicate positions.
 """
 
-import sys
 import pysam
 import logging
-
-import blr.utils as BLR
+from tqdm import tqdm
 
 logger = logging.getLogger(__name__)
 
 
 def main(args):
+    logger.info("Starting Analysis")
+
+    summary = Summary()
+
+    current_cache_rp = dict()
+    cache_read_pair_tracker = CacheReadPairTracker(None, None, None, None, None)
```

Comment:
That's true, it was needed in a previous version of the rewrite, I'll remove it as well as the `if read and mate` statement in the init function since this was only written to not run when read and mate was supplied as `None`.